src/m365/purview/commands/retentionevent/retentionevent-get.ts

```diff
@@ -0,0 +1,75 @@
+import { Logger } from '../../../../cli/Logger';
+import GraphCommand from '../../../base/GraphCommand';
+import GlobalOptions from '../../../../GlobalOptions';
+import commands from '../../commands';
+import request, { CliRequestOptions } from '../../../../request';
+import { validation } from '../../../../utils/validation';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+}
+
+class PurviewRetentionEventGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.RETENTIONEVENT_GET;
+  }
+
+  public get description(): string {
+    return 'Retrieve the specified retention event';
+  }
+
+  constructor() {
+    super();
+
+    this.#initOptions();
+    this.#initValidators();
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-i, --id <id>'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (!validation.isValidGuid(args.options.id)) {
+          return `'${args.options.id}' is not a valid GUID.`;
+        }
+
+        return true;
+      }
+    );
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      if (this.verbose) {
```

Comment:
Could you add a check for the command running in delegated mode. If not it should throw an error.